src/Todo.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import InputBox from './InputBox';
+import TodoList from './TodoList';
```

Comment:
```suggestion
import React from 'react';

import InputBox from './InputBox';
import TodoList from './TodoList';
```

공백 추가해주세요~ 모듈 간에 역할이 다르기 때문에 관심사를 분리하는 것 처럼 구분해두는 습관을 가지면 좋습니다~ 추후 임포트하는게 많아질 경우 편리하구요 :)

다른 파일들도 동일하게 처리해주세요~